sympy/physics/vector/tests/test_frame.py

```diff
@@ -158,6 +158,23 @@ def test_dcm():
          cos(q1)*cos(q2)]])
 
 
+def test_w_diff_dcm():
```

Comment:
Please add a comment explaining what bug this is testing and a reference issue where it was discovered.

Reply:
w_diff_dcm gives results which don't match with the Autolev ones so I am assuming they are incorrect. Also, there were no tests for this so I think it wasn't checked properly if it works.
I am not sure why my change is correct but it seems to work. Should I open an issue for it?

Reply:
I just want some explanation in the code so that future authors can understand what happened.